packages/vuepress-theme-odyssey/components/DocsNav.vue

```diff
@@ -26,6 +26,7 @@
               'docs-nav-link': true,
               'is-docs-nav-link--active': isCurrentRoute(item.link)
             }"
+            target="_self"
```

Comment:
Just make this as the default? Why do we have to explicitly call it out?

Reply:
by default external `DocsLinks` have target `_blank`, this was an instance where we explicitly did not want that! Good catch.

Reply:
Got it, I still assume that we want new pages for doc links in majority of the cases? This is just an exception right?